scripts/rmc_flags.py

```diff
@@ -82,6 +82,9 @@ def add_artifact_flags(make_group, add_flag, config):
              help="Generate a goto symbol table")
     add_flag(group, "--keep-temps", default=False, action=BooleanOptionalAction,
              help="Keep temporary files generated throughout RMC process")
+    # False => flag not present; None => flag present, no argument provided
+    add_flag(group, "--save-logs", nargs="?", type=pl.Path, default=False,
+             help="Save RMC logs to <target_dir>/rmc.log, or provide a file to save logs to.")
```

Comment:
Can you please state the default behavior?

Reply:
Done!